resolwe_bio/tools/bed_file_corrections_genome_browsers.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+"""Maximize 5th column of bed file to 1000."""
```

Comment:
I would rephrase this. To me, "maximize" means that something is **raised** (to the full potential). But if i see correctly, you are actually **lowering** the scores to be below 1000.  I would rather see something like "Restrict all scores to the maximal value of `LIMIT`." And set a constant `LIMIT=1000`.